Gruntfile.js

```diff
@@ -87,7 +87,11 @@ module.exports = function(grunt) {
         options: {
           jshintrc: '.jshintrc'
         },
-        src: 'src/js/**/*.js'
+        src: [
+          'src/js/**/*.js',
+          '!src/js/extensions/jquery.js',
+          '!src/js/extensions/jquery.once.js'
```

Comment:
For external libraries, we should keep them in a separate folder like `lib`, not `src` (which is for this extension).